concepts/pipeline/event_pipeline.md

```diff
@@ -0,0 +1,127 @@
+# Event Data Pipeline
+
+We collect event-oriented data from different sources. This data is collected and processed in a
+specific path through our data pipeline, which we will detail here.
+
+```mermaid
+graph TD
+fx_code(fa:fa-cog Firefox code) --> firefox(fa:fa-firefox Firefox Telemetry)
+fx_extensions(fa:fa-cog Mozilla extensions) --> firefox
+fx_hybrid(fa:fa-cog Hybrid Content) --> firefox
+mobile(fa:fa-cog Mobile products) --> mobile_telemetry(fa:fa-firefox Mobile Telemetry)
+firefox -->|main ping| pipeline((fa:fa-database Firefox Data Pipeline))
+firefox -.->|events ping, planned| pipeline
+mobile_telemetry --> |mobile events ping| pipeline
+pipeline -->|Firefox events| main_summary[fa:fa-bars main summary table]
+main_summary --> events_table[fa:fa-bars events table]
+pipeline -->|Mobile events| focus_events_table[fa:fa-bars mobile events table]
+main_summary --> redash(fa:fa-bar-chart Redash)
+events_table --> redash
+events_table -.->|planned| amplitude(fa:fa-bar-chart Amplitude)
+focus_events_table --> redash
+focus_events_table --> amplitude
+
+style fx_code fill:#f94,stroke-width:0px
+style fx_extensions fill:#f94,stroke-width:0px
+style fx_hybrid fill:#f94,stroke-width:0px
+style mobile fill:#f94,stroke-width:0px
+style firefox fill:#f61,stroke-width:0px
+style mobile_telemetry fill:#f61,stroke-width:0px
+style pipeline fill:#79d,stroke-width:0px
+style main_summary fill:lightblue,stroke-width:0px
+style events_table fill:lightblue,stroke-width:0px
+style focus_events_table fill:lightblue,stroke-width:0px
+style redash fill:salmon,stroke-width:0px
+style amplitude fill:salmon,stroke-width:0px
+```
+
+# Overview
+
+Across the different Firefox teams there is a common need for a more fine grained understanding of
+product usage, like understanding the order of interactions or how they occur over time.
+To address that our data pipeline needs to support working with event-oriented data.
+
+We specify a common event data format, which allows for broader, shared usage of data processing tools.
+To make working with event data feasible, we provide different mechanisms to get the event data
+from products to our data pipeline and make the data available in tools for analysis.
+
+# The event format
```

Comment:
Should we provide a link to the corresponding section in the source docs in this section as well?